src/test/java/uk/gov/justice/hmpps/prison/executablespecification/KeyworkerStepDefinitions.java

```diff
@@ -74,7 +74,7 @@ public void aKeyWorkerAllocationsRequestIsMadeWithNomisIdsAndAgency(final String
 
     @When("^a key worker allocation history request is made with staff ids \"([^\"]*)\"$")
     public void aKeyWorkerAllocationHistoryRequestIsMadeWithStaffIdsAndAgency(final String staffIds) throws Throwable {
-        keyworker.getKeyworkerAllocationHistoryByStaffIds(Arrays.stream(StringUtils.split(staffIds, ",")).map(Long::new).collect(Collectors.toList()));
+        keyworker.getKeyworkerAllocationHistoryByStaffIds(Arrays.stream(StringUtils.split(staffIds, ",")).map(Long::valueOf).collect(Collectors.toList()));
```

Comment:
Unrelated non-functional change.  I noticed it what showing a deprecation warning when running the test suite, this resolves that.